Documentation/installation/k8s-install-helm.rst

```diff
@@ -192,11 +192,17 @@ Install Cilium
 
           Cilium can alternatively run in EKS using an overlay mode that gives
           pods non-VPC-routable IPs.  This allows running more pods per
-          Kubernetes worker node than the ENI limit, but means that pod
-          connectivity to resources outside the cluster (e.g., VMs in the VPC
-          or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use
-          the VPC IP address of the Kubernetes worker node. To set up Cilium 
-          overlay mode, follow the steps below:
+          Kubernetes worker node than the ENI limit but includes the following caveats:
+
+            1. Pod connectivity to resources outside the cluster (e.g., VMs in the VPC
+               or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use the
+               VPC IP address of the Kubernetes worker node.
+            2. The EKS API Server is unable to route packets to the overlay network. This
```

Comment:
Minor nit while looking through afterwards, I think that unordered lists would be more appropriate here and would help to break up the flow of the document vs. the ordered list below. I don't think that there is any specific meaning to the order of these caveats, they're each unique points that the user may consider.